UM/PackageManager.py

```diff
@@ -71,6 +71,7 @@ def __init__(self, application: "QtApplication", parent: Optional[QObject] = Non
         self._to_remove_package_set = set()  # type: Set[str] # A set of packages that need to be removed at the next start
         self._to_install_package_dict = {}  # type: Dict[str, Dict[str, Any]]  # A dict of packages that need to be installed at the next start
         self._dismissed_packages = set()    # type: Set[str] # A set of packages that are dismissed by the user
+        self.installed_packages = {}  # type: Dict[str, Dict[str, Any]]  # A dict of packages that were installed during startup
```

Comment:
This should be protected. Only the package manager should be adding packages to this dict, right?